app/src/main/java/com/andela/omokolataiwo/levelup/presenter/GithubProfilePresenter.java

```diff
@@ -21,36 +23,41 @@
      */
     private final MainContract.MainView mView;
 
+    /**
+     * Coordinator layout.
+     */
+    private final CoordinatorLayout coordinatorLayout;
+
     /**
      * GithubProfilePresenter constructor.
      *
-     * @param view Activity of the request.
+     * @param view              Activity of the request.
+     * @param coordinatorLayout coordinator layout
      */
-    public GithubProfilePresenter(MainContract.MainView view) {
+    public GithubProfilePresenter(MainContract.MainView view, CoordinatorLayout coordinatorLayout) {
```

Comment:
@omokolataiwo 
Once you are able to extract the `Snackbar` implementation to the activity itself, you will no longer require to pass `CoordinatorLayout` as a  constructor parameter.... As a matter of standards, always make sure you never have anything that has to do with `Android` stuff, be present in your `Presenter` classes... Use the `contract interface` to achieve that.